conf/modules.config

```diff
@@ -198,5 +198,13 @@ process {
         ]
         ext.prefix = { "${report}" }
     }
+
+        withName: GECCO_RUN {
+        publishDir = [
+            path: { "${params.outdir}/gecco/run/" },
```

Comment:
The module is called gecco/run, isn't it more clear if we keep the structure like this? Now that you mention it, I see that for deeparg, macrel and hmmer you changed this into:
- `"${params.outdir}/deeparg/"` for deeparg/downloaddata
- `"${params.outdir}/macrel"` for macrel/contigs
- `"${params.outdir}/hmmer_hmmsearch"` for hmmer/hmmsearch
What is the advantage? I only see that it might be a little confusing if the submodule directories are renamed. But not so confusing that I really bother. I just want your opinion and then decide for one or the other folder structure.

Reply:
Alright. As we don't plan to include other gecco modules like 
![image](https://user-images.githubusercontent.com/73216762/180800845-bc97b238-6b50-4114-9136-e0f39e9da56d.png)
(of which only `run` exists until now), let's use the single-folder way.

Reply:
```suggestion
            path: { "${params.outdir}/gecco/" },
```